<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiger Hash</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_msg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>明文</string>
    </property>
    <widget class="QPlainTextEdit" name="msgText">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>731</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_hash">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>249</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>hash值</string>
    </property>
    <widget class="QPlainTextEdit" name="hashText">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_mode">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>249</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>模式选择</string>
    </property>
    <widget class="QRadioButton" name="RB_Tiger">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiger</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RB_Tiger2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiger2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="PB_start">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>253</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计 算</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
